qdb/comm.py

```diff
@@ -58,12 +58,18 @@ def fmt_msg(event, payload=None, serial=None):
     Serial is a function to call on the frame to serialize it, e.g:
     json.dumps.
     """
+    if event=="locals":
```

Comment:
I think it should be the responsiblity of the caller to format the payload correctly, we should try to alter the `locals` command code instead of the send `fmt_msg` code.